### ISSUE TITLE
COTD overall best disappearing

### ISSUE THREAD

**Issue by racacax** [openplanet-nl/tmio#20]
Hi,
I noticed on the player COTD tab that as soon as the player wins a div 1 COTD (primary or rerun), the "Overall best" container disappears.
Is it the expected behaviour ? 
Example here : https://trackmania.io/#/player/84078894-bae1-4399-b869-7b42a5240f02/cotd
I won a rerun, and now there is only the Primary container.

Also, I noticed that when you have an equally best div but a later better round result in a rerun, the date displayed in the overall tab is the one from the primary one.
Example : You finished 20th in div 1 in the main COTD on the 1st of March. This is your best result.
You finished 3rd in div 1 in a rerun on the 1st of June. This is your best result for a rerun.
The date shown on the rerun will be "4 months ago" instead of "a month ago"

**Comment by codecat** [comment]
Yes, that would be the expected behavior, because it's just going to say "1st" for every value anyway, which wouldn't be that useful to know. I guess it does show the time when the rank was obtained though. When I made this page, I figured it wouldn't add much value to show it in that case.

The date shown being the first time you did it is also intentional. Since technically if you equaled your best rank it's probably more interesting to show when you first did it rather than the last time you did it.

Do you think either of these should be changed? :thinking:

**Comment by racacax** [comment]
Actually, it would not display 1st for each one in my case. On the primary one, I would have 21st as best (as its currently written) and 1st on Overall best (which is hidden atm). But it makes sense to hide the overall one when someone won in the primary. So I believe displaying "Overall best" only when its better than "Primary best" should be the correct behaviour.

Ah ok I understand. So currently, the date is only for the div not for div + placement. I thought it was for the combo of the 2. But both make sense actually. Its either saying "Your best division is 1, achieved for the first time a year ago. Your best result is 22nd" (current behaviour) or "Your best result is 22nd in div 1 achieved for the first time a year ago". So I don't really know about this one 🤔
